torch/distributions/wishart.py

```diff
@@ -56,6 +61,7 @@ class Wishart(ExponentialFamily):
     }
     support = constraints.positive_definite
     has_rsample = True
+    _mean_carrier_measure = 0
```

Comment:
`_mean_carrier_measure` is mendatory to inherit `torch.distributions.exp_family.ExponentialFamily`.
Without the assigment of `_mean_carrier_measure`, every test about exponential family does not triggered.
It is my bad I missed this part before.